nservicebus/azure-service-bus/configuration/azureservicebusqueueconfig.md

```diff
@@ -35,6 +35,6 @@ NOTE: Most of these values are applied when a queue or topic is created and cann
  * `EnablePartitioning`: Increase overall throughput by allowing to use multiple brokers/stores to handle queues and topics to overcome limitations of a single broker/store at increased cost. Partitioning does reduce number of queues or topics per namespace. Defaults to false.
  * `SupportOrdering`: Specifies whether queues and topics should enable support for message ordering. Defaults to true.
 
-NOTE: `QueueName` and `QueuePerInstance` are obsoleted. Instead, use bus configuration object to specify endpoint name and scale-out option.
+NOTE: AzureServiceBusQueueConfig is only available in Version 6 and below. `QueueName` and `QueuePerInstance` are obsoleted since version 6. Instead, use bus configuration object to specify endpoint name and scale-out option.
```

Comment:
Should this be endpoint configuration rather than bus configuration?

Are QueueName and QueuePerInstance related to AzureServiceBusQueueConfig? The first two sentences say one is available in v6 and below AND that they are obsolete in v6. If these are not related, do they belong in the same note?